kanalony-storage-logic/src/main/scala/kanalony/storage/logic/Queries.scala

```diff
@@ -9,9 +9,11 @@ object Queries {
   object HourlyUserActivityPrtnEntryClstCountry extends HourlyUserActivityPrtnEntryClstCountryQuery
   object HourlyUserActivityPrtnEntryQuery extends HourlyUserActivityPrtnEntryQuery
   object HourlyUserActivityClstEntryQuery extends HourlyUserActivityClstEntryQuery
+  object DailyUserActivityPrtnEntryQuery extends DailyUserActivityPrtnEntryQuery
   val queries : List[IQuery] = List(
```

Comment:
We will have to split this list into smaller branches, otherwise we'll find ourselves with hundreds of items in this list.